pkg/osd/cluster.go

```diff
@@ -120,19 +125,97 @@ func (u *OSD) DeleteCluster(clusterID string) error {
 }
 
 // WaitForClusterReady blocks until clusterID is ready or a number of retries has been attempted.
-func (u *OSD) WaitForClusterReady(clusterID string, timeout time.Duration) error {
+func (u *OSD) WaitForClusterReady(cfg *config.Config, clusterID string, timeout time.Duration) error {
 	log.Printf("Waiting %v for cluster '%s' to be ready...\n", timeout, clusterID)
+	cleanRuns := 0
 
 	return wait.PollImmediate(45*time.Second, timeout, func() (bool, error) {
```

Comment:
I'll switch this to 30s. It might make the logs somewhat chattier during install but it will resolve this faster AND give us more granularity on events.